contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -177,7 +177,7 @@ def __sub__(self, other: 'Millisatoshi') -> 'Millisatoshi':
         return Millisatoshi(int(self) - int(other))
 
     def __mul__(self, other: int) -> 'Millisatoshi':
-        return Millisatoshi(self.millisatoshis * other)
+        return Millisatoshi(int(self.millisatoshis * other))
```

Comment:
Could also use `math.floor()` to make rounding down more apparent.

Reply:
Should I replace all the 'int' casts with that? (__truediv__, __foordiv__, ... ) etc?